packages/backend/src/api/request.ts

```diff
@@ -16,6 +16,7 @@ export type ClerkBackendApiRequestOptions = {
   queryParams?: Record<string, unknown>;
   headerParams?: Record<string, string>;
   bodyParams?: object;
+  formData?: FormData;
```

Comment:
:question: Forgive me for the newbie question, but is `FormData` available in all target runtimes we want to support?

Reply:
This is actually a spot-on question. Indeed, `FormData` is available in node 18.x and above, and all the runtimes that support the Fetch API.

I'll be adding a complementary change when https://github.com/clerkinc/javascript/pull/1393 gets merged, which adds `FormData` support for node versions prior to 18.

Reply:
@nikosdouvlis  We will use it when #1393 gets merged, which re-exports `FormData` from the pollyfill.

Reply:
Just added support for Node prior to 18.x versions.